lte/protos/oai/mme_nas_state.proto

```diff
@@ -132,6 +132,10 @@ message UeContext {
   uint32 granted_service = 100; // granted_service_t
   uint32 cs_fallback_indicator = 101;
   SgsContext sgs_context = 102;
+  uint32 tau_accept_eps_ber_cntx_status = 103;
+  uint32 nb_delete_sessions = 104;
+  uint32 nb_delete_bearer_cmd = 105;
+  bool mme_initiated_ded_bearer_deactivation = 106;
```

Comment:
are these supposed to be serialized/deserialized for stateless? cc @ssanadhya

Reply:
Yes.
@pruthvihebbani , can we add the code in MME/NAS state converters and also add an S1ap test for MME restart?

Reply:
I have added state conversion in mme_app_state_converter.cpp. I will raise a separate PR for restart TC

Reply:
These changes look good, thanks!